newIDE/app/src/EventsSheet/index.js

```diff
@@ -2140,6 +2152,17 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
                     }
                   />
                 )}
+                {this.state.layoutVariablesDialogOpen && (
+                  <GlobalAndSceneVariablesDialog
+                    projectScopedContainersAccessor={
+                      projectScopedContainersAccessor
+                    }
+                    open
+                    onCancel={() => this.editLayoutVariables(false)}
+                    onApply={() => this.editLayoutVariables(false)}
```

Comment:
there's no need for a specific onApply action here?

Reply:
I guess the state change to close the dialog already trigger a refresh of all events.
I tried to use an unknown variable and then add it. When I close the dialog, it becomes blue.